gibberish_client/src/components/Gameroom/QuestionCardComponent.js

```diff
@@ -8,46 +8,33 @@ import GameWaiting from './gamestates/GameWaiting'
 import RoomDetailComponent from './RoomDetailComponent'
 
 function QuestionCardComponent(props) {
-  const { roomId, currentRound, maxRounds, gamestate, roundScores, players, startNextRound, transitionToState, updateScores, currentQuestion, currentAnswer, timeRemaining, setTimeRemaining  } = props
+  const { roomId, currentRound, gamestate, players, updateScores, qna, timeRemaining  } = props
 
   let component
   if(gamestate === gamestates.ROUND_LOADING) {
     component = <RoundLoading 
-                  transitionToState={transitionToState} 
-                  nextState={gamestates.ROUND_ONGOING}
-                  timeRemaining={timeRemaining}
-                  setTimeRemaining={setTimeRemaining}/>
+                  timeRemaining={timeRemaining}/>
   } else if(gamestate === gamestates.ROUND_ONGOING) {
     component = <RoundOngoing 
-                  transitionToState={transitionToState} 
-                  nextState={gamestates.ROUND_ENDED}
                   updateScores={updateScores}
-                  currentQuestion={currentQuestion} 
-                  currentAnswer={currentAnswer} 
-                  timeRemaining={timeRemaining}
-                  setTimeRemaining={setTimeRemaining}/>
+                  currentQuestion={qna[currentRound-1]['question']} 
```

Comment:
Would need to update this if we are using a set

Can possibly try one of the suggestions here!
https://stackoverflow.com/questions/44715210/how-to-get-index-based-value-from-set/44715657

Reply:
resolved by converting set to array